addons/io/org.openhab.io.homekit/.classpath

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8"/>
+	<classpathentry kind="con" path="org.eclipse.pde.core.requiredPlugins"/>
+	<classpathentry kind="src" path="src/main/java"/>
+	<classpathentry exported="true" kind="lib" path="lib/com.beowulfe.hap.hap-1.1.2.jar"/>
```

Comment:
Could you maybe add an about.html similar to https://github.com/openhab/openhab2-addons/blob/master/addons/binding/org.openhab.binding.freebox/about.html#L28, which lists all included 3rd party libs and mentions their licenses?